printrun/printcore.py

```diff
@@ -13,7 +13,7 @@
 # You should have received a copy of the GNU General Public License
 # along with Printrun.  If not, see <http://www.gnu.org/licenses/>.
 
-__version__ = "1.6.0"
+__version__ = "2.0.0rc1"
```

Comment:
Would it be much trouble if it was "2.0.0-rc1" to comply with [semver](https://semver.org/)?

Reply:
I went with [PEP 440](https://www.python.org/dev/peps/pep-0440/).

> Semantic versions containing a hyphen (pre-releases - clause 10) or a plus sign (builds - clause 11) are not compatible with this PEP and are not permitted in the public version field.

Reply:
Oh sorry, ok. I wasn't aware of this incompatibility.